### PR TITLE
log http api requests with zipkin headers

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

File: vit-servicing-station-lib/src/v0/mod.rs [hunk 1]
```diff
@@ -16,9 +16,33 @@ pub async fn filter(
...
+            method = %info.method(),
+            path = info.path(),
+            version = ?info.version(),
+            remote_addr = Empty,
+            trace_id = Empty,
+            span_id = Empty,
+            parent_span_id = Empty,
+        );
+        if let Some(remote_addr) = info.remote_addr() {
+            span.record("remote_addr", &remote_addr.to_string().as_str());
```
Comment (mzabaluev) [inline vit-servicing-station-lib/src/v0/mod.rs:34]: Is this going to be the address of the load balancer in our infrastructure?

**Review by mzabaluev — APPROVED** [review]

File: vit-servicing-station-lib/src/v0/mod.rs (hunk 1 above)
Comment (danielSanchezQ) [reply vit-servicing-station-lib/src/v0/mod.rs:34]: I think is the remote address making the request. It is saved in the `remote_addr` record attribute

**Review by danielSanchezQ — APPROVED** [review]
LGTM

File: vit-servicing-station-lib/src/v0/mod.rs (hunk 1 above)
Comment (mzabaluev) [reply vit-servicing-station-lib/src/v0/mod.rs:34]: I'm not sure this is what warp puts into the `info` structure. Or does it parse the headers set up by the proxy to get at the original source IP address?

**Pull request merged**